### PR TITLE
github: Add issue template

### LINKED ISSUE #186: Add github issue template
Copy https://github.com/kata-containers/runtime/blob/master/.github/ISSUE_TEMPLATE.md into this repo.

### PR DESCRIPTION
**Pull request by jodh-intel**
Add a template to guide the user on raising github issues.

Fixes #186.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

File: grpc.go [hunk 1]
```diff
@@ -665,6 +665,8 @@ func (a *agentGRPC) TtyWinResize(ctx context.Context, req *pb.TtyWinResizeReques
 }
 
 func (a *agentGRPC) CreateSandbox(ctx context.Context, req *pb.CreateSandboxRequest) (*gpb.Empty, error) {
+	agentLog.WithField("grpc-request", req).Debug()
```
Comment (sboeuf) [inline grpc.go]: This should not be part of this PR, right ??

Comment (jodh-intel) [reply grpc.go]: Oops! Updated ;)

**Comment by devimc** [comment]
lgtm

**Comment by jodh-intel** [comment]
@klynnrif - ptal.
